packages/botonic-core/package.json

```diff
@@ -19,6 +19,23 @@
   "dependencies": {
     "axios": "^0.19.0"
   },
-  "devDependencies": {},
-  "homepage": "https://github.com/hubtype/botonic#readme"
+  "devDependencies": {
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/plugin-transform-runtime": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/register": "^7.4.0",
+    "@babel/runtime": "^7.4.3",
+    "babel-plugin-add-module-exports": "^1.0.0",
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-no-null": "^1.0.2",
+    "eslint-plugin-prettier": "^3.0.1",
+    "prettier": "^1.17.0"
+  },
+  "homepage": "https://github.com/hubtype/botonic#readme",
+  "engines": {
+    "node": ">=8.10.0"
+  }
```

Comment:
Are all this dependencies really needed in core?

Reply:
they're just dev dependencies, mostly for babel & linters. Since we don't even really need to build core/react,  I think they're harmless